packages/terra-tabs/src/Tabs.module.scss

```diff
@@ -91,6 +91,15 @@
       font-size: 0.571rem;
       margin-left: var(--terra-tabs-menu-arrow-margin-left, 0.5rem);
     }
+
+    .active-tab-icon {  
+      margin-right: var(--terra-tabs-menu-icon-margin-right, 0.5rem);
+  
+      /* stylelint-disable-next-line selector-max-compound-selectors */
+      svg {
+        font-size: var(--terra-tabs-menu-icon-font-size, 1rem);
```

Comment:
Let's add a note here to update fusion-theme and low-light-theme for these new variables 
- --terra-tabs-menu-icon-margin-right
- --terra-tabs-menu-icon-font-size